nodeprobe/nodeprobe.go

```diff
@@ -109,7 +110,12 @@ func (p *Prober) probe(ctx context.Context) {
 	p.healthy.Store(healthy.Load())
 
 	if !p.healthy.Load() {
-		p.logger.Error("not all nodes are healthy")
+		p.logger.Error(fmt.Sprintf(
```

Comment:
probably zap fields would be a bit more readable, but I don't mind leaving this as is

Reply:
Changed to
```go
		p.logger.Error(
			"not all nodes are healthy",
			zap.Strings("nodes", []string{EventSyncerNode, ConsensusClientNode, ExecutionClientNode}),
		)
```